docs/source/user-guide/tasks/work-with-packages.rst

```diff
@@ -13,19 +44,136 @@ environment:
    the packages that you want to include. You can specify as many
    packages as you want.
 
-   The packages are installed in your project's environment, so
-   you now see package files in your project folder, such as::
+   EXAMPLE: To add the packages hvplot=0.7 and dask::
 
-     envs/PATH/package1
+     $ anaconda-project add-packages hvplot=0.7 dask
+     Collecting package metadata (current_repodata.json): ...working... done
+     Solving environment: ...working... done
+     ...
+     Executing transaction: ...working... done
+     Using Conda environment /Users/adefusco/Desktop/iris/envs/default.
+     Added packages to project file: hvplot=0.7, dask.
 
-   NOTE: Replace PATH with the actual path to your package.
+#. OPTIONAL: In a text editor, open ``anaconda-project.yml`` to
+   see the new packages listed in the packages section. To install the packages
+   run ``anaconda-project prepare``.
 
-   EXAMPLE: To add the packages Bokeh and pandas::
+*****************
+Removing packages
+*****************
 
-     $ anaconda-project add-packages bokeh=0.12 pandas
-     conda install: Using Anaconda Cloud api site https://api.anaconda.org
-     Using Conda environment /home/alice/mystuff/iris/envs/default.
-     Added packages to project file: bokeh=0.12, pandas.
+To remove packages from the ``packages:`` list run::
 
-#. OPTIONAL: In a text editor, open ``anaconda-project.yml`` to
-   see the new packages listed in the packages section.
+  anaconda-project remove-packages package1 package2
+
+NOTE: Replace ``package1`` and ``package2`` with the names of
+the packages that you want to include. You can specify as many
+packages as you want.
+
+EXAMPLE: To remove the package hvplot::
+
+  $ anaconda-project remove-packages hvplot
+  Using Conda environment /Users/adefusco/Desktop/testproj/envs/default.
+  Removed packages from project file: hvplot.
+
+******************
+Using pip packages
+******************
+
+In addition to adding and removing Conda packages as shown above Pip packages
+can be specified using the ``--pip`` flag::
+
+  anaconda-project add-packages --pip package1 package2
+  anaconda-project remove-packages --pip package1 package2
+
+NOTE: Replace ``package1`` and ``package2`` with the names of
+the packages that you want to include. You can specify as many
+packages as you want.
+
+EXAMPLE: To add the ``requests`` package to the default environment::
```

Comment:
Shouldn't this be an example of _removing_ a pip package?

Reply:
I've restructured this page a bit to make it more clear